generator/util/query.py

```diff
@@ -300,55 +299,72 @@ def get_parent_class(class_object):
 
     .. code-block:: xml
 
-       <element name="ArrayChild" typeCode="SBML_TEST_ARRAYCHILD" hasListOf="false" hasChildren="false" hasMath="false" childrenOverwriteElementName="false" baseClass="SBase" abstract="false" elementName="arrayChild">
+       <element name="ArrayChild" typeCode="SBML_TEST_ARRAYCHILD"
+                hasListOf="false" ... baseClass="SBase" abstract="false"
+                elementName="arrayChild">
           <attributes>
-             <attribute name="unit" required="false" type="element" element="Unit" abstract="false"/>
+             <attribute name="unit" required="false" type="element"
+                        element="Unit" abstract="false"/>
           </attributes>
        </element>
-    """
-    parent = ''
+
+    e.g. in the following XML, the 'Def' element is reerenced by the
+     'Model' plugin
+    so
+
+    .. code-block:: default
+
+       get_parent_class(class_object_for Def)
+
+    would return
+
+    .. code-block:: default
+
+       'Model'
+
+    .. code-block:: xml
+
+      <elements>
```

Comment:
I think this XML block needs to be indented another space or two, or it won't be nicely formatted in Sphinx.

Reply:
It must be indented at least as far as the 'c' at the start of 'code-block'